domains/transfer-agreements/TransferAgreements.API/API/Data/TransferAgreementRepository.cs

```diff
@@ -35,4 +35,21 @@ public async Task<List<TransferAgreement>> GetTransferAgreementsList(Guid subjec
             .Where(agreement => agreement.Id == id && (agreement.SenderId == Guid.Parse(subject) || agreement.ReceiverTin.Equals(tin)))
             .FirstOrDefaultAsync();
     }
+
+    public async Task Save()
```

Comment:
Why this method?

Reply:
Because some other developers wanted a separate method, for only saving.